docs/new_instrument.rst

```diff
@@ -268,8 +270,8 @@ See pysat.utils.time.create_datetime_index for creating a datetime index for an
 array of irregularly sampled times.
 
 pysat will invoke the list_files method the first time a particular instrument
-is instantiated. After the first instantiation, by default pysat will not search
-for instrument files as some missions can produce a large number of
+is instantiated. After the first instantiation, by default, pysat will not 
+search for instrument files as some missions can produce a large number of
 files which may take time to identify. The list of files associated
```

Comment:
```suggestion
search for instrument files as some missions can produce a large number of
files that may take time to identify. The list of files associated
```

Reply:
I think that "which" is correct here because the clause "may take time to identify" can be removed from the sentence without breaking it.  We could put a comma after files to make that more clear if we wanted to.

Reply:
According to the Elements of Style, either is appropriate depending on the phrasing.  If you would like to use "which", it needs to be preceded by a comma.